code/game/objects/items/weapons/tools.dm

```diff
@@ -721,6 +721,55 @@
 	playsound(get_turf(src), 'sound/medbot/Flatline_custom.ogg', 35)
 	return (SUICIDE_ACT_BRUTELOSS)
 
+/obj/item/tool/crowbar/halligan
+	name = "Halligan bar"
+	desc = "Combination pick, crowbar, and adze used for forcible entry."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "halligan"
+	hitsound = "sound/weapons/toolhit.ogg"
+	item_state = "halligan"
+	w_class = W_CLASS_MEDIUM
+	attack_verb = list("pries", "slashes", "stabs", "bludgeons", "whacks")
+
+/obj/item/tool/crowbar/halligan/attackby(obj/item/I, mob/user)
+	if(istype(I,/obj/item/weapon/fireaxe))
+		var/obj/item/weapon/fireaxe/F = I
+		to_chat(user, "<span class='notice'>You attach the fireaxe and Halligan bar to carry them easier.</span>")
```

Comment:
Its best to use
`You attach [F] and [src] to carry them easier.`
Instead of writing down the names in the message. It lets things like labels or custom names get shown in the message.